theme-utils.mjs

```diff
@@ -35,6 +36,24 @@ function showHelp(){
 	console.log('Help info can go here');
 }
 
+/*
+ Update version number in style.css and style-child-theme.css for each theme
+ after version bump from conventional commits (via Lerna)
+ Used by Lerna via update-stylecss command
+*/
+async function versionBumpStyleCss() {
```

Comment:
Doesn't version bumping resources based on package.json values rather than using style.css as the source-of-truth require all themes to leverage a package.json file?

Reply:
True. If we can use style.css that's better.

Reply:
Yeah that's right, it means all themes would need a package.json file. That's how Lerna works out of the box, but I'll see if I can find a good way to make style.css the source of truth instead.